railties/lib/rails/tasks/dev.rake

```diff
@@ -1,10 +1,11 @@
 # frozen_string_literal: true
 
-require "rails/dev_caching"
+require "rails/command"
```

Comment:
We should probably do `require "active_support/deprecation"` here as well.

Reply:
Done :)